src/components/Editor/Tabs.js

```diff
@@ -98,7 +97,7 @@ type Props = {
   startPanelCollapsed: boolean,
   endPanelCollapsed: boolean,
   searchOn: boolean,
-  sourceMetaData: SourceMetaDataMap
+  sourceTabsMetaData: any
```

Comment:
hmmm perhaps we can make a type annotation?

Reply:
Roger that